src/app.types.tsx

```diff
@@ -94,8 +94,33 @@ export interface AddCatalogueCategoryProperty {
   unit?: string;
   mandatory: boolean;
   allowed_values?: AllowedValues;
+  default_value?: string | number | boolean;
```

Comment:
I would rather we separate out form schemas and backend api schemas at some point as I find this a bit confusing, but am happy for us to do that later. In this case we could use a union instead for the two types rather than having many optional fields to cater for both.

On the backend we call them `CatalogueItemPropertyPostRequestSchema` and `CatalogueCategoryPostRequestPropertySchema` although the names are going to change.